protobuf/go.mod

```diff
@@ -4,5 +4,5 @@ go 1.16
 
 require (
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
-	google.golang.org/protobuf v1.26.0
+	google.golang.org/protobuf v1.5.0
```

Comment:
did we downgrade? what's the reason for this here?

Reply:
flow-go-sdk uses even older `1.4.x` and there were compatibility and compilation issues. I will check maybe if we can bump `flow-go-sdk` instead.